action.yml

```diff
@@ -290,3 +290,39 @@ runs:
       env:
         # this field would set the original repo command mount command (would be mounted always in the old flow) and only if checkout = true in the new flow
         mount_original_repo_command: ${{ fromJSON('["","-v $(pwd)/code:/code"]')[((!(fromJSON(github.event.inputs.client_payload).payload.runner_setup) || (fromJSON(github.event.inputs.client_payload).payload.runner_setup.checkout == true)))] }}
+
+    - name: Temp Raise Error
+      if: always()
+      run: exit 1
+      shell: bash
+
+    - name: Notify Jit on Failure
+      if: failure()
+      uses: actions/github-script@v6
+      with:
+        script: |
+          console.log(`Notifying Jit of failure`);
+          const headers = {
+            'Authorization': `Bearer ${{ steps.oidc-auth.outputs.jit_token }}`,
```

Comment:
we won't always have oidc token
we should use `${{ fromJSON(github.event.inputs.client_payload).jit_jwt_token`